### ISSUE TITLE
Increase fps with less accuracy

### ISSUE THREAD

**Issue by mirkow** [danielgordon10/re3-tensorflow#2]
First, great tracker!

I run it on a raspberry pi 4, but there I only get 2.5 FPS.
Is there a way to easily increase the FPS but maybe with less accuracy as a drawback?

Thanks!

**Comment by mirkow** [comment]
Found now https://github.com/danielgordon10/re3-pytorch, which is supposed to have a smaller model..I'll try it and update with results

**Comment by mirkow** [comment]
Update: The small model in the pytorch variant is even slower than the big model in the tensorflow variant (only 1.2 fps).

**Comment by Rajasekhar06** [comment]
How did you ran the tracker only demo_batch is working for me

**Comment by mirkow** [comment]
I could just execute it. But this issue is probably not the place to discuss that

**Comment by danielgordon10** [comment]
I don't think there's many deep learning models you can run on a Raspberry Pi. You could maybe try a MobileNet/Shufflenet backbone. I don't see why they wouldn't work. But adding these is not something I plan on doing.
